### DISCUSSION TITLE
GitHub Profile Overview Year filter is Weird

### DISCUSSION THREAD

**Discussion by Yodawgz0** [community/community#109177] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When you select a year, such as 2022, the URL changes to reflect the current year’s state. It then applies a filter, allowing you to view contributions made during that specific year. 

However, the issue arises when you click the ‘back’ button—the browser navigates to the previous URL (e.g., from https://github.com/Yodawgz0?tab=overview&from=2023-12-01&to=2023-12-31 to https://github.com/Yodawgz0). 

Despite this change in URL, the UI remains unaffected, and the filter persists. To reset the profile overview to the default view, you’ll need to manually refresh the page

**Comment by Shahnawazk7944** [comment]
I have just noticed, you're right. I think it's a GitHub feature 😊 which is intentionally added. It's a nice touch from GitHub to add these little details for users to discover and enjoy.

**Reply by Yodawgz0** [reply]
I see yeah it could be that too
